### DISCUSSION TITLE
A testing error

### DISCUSSION THREAD

**Discussion by Am121000** [pykt-team/pykt-toolkit#73] [General]
We have been testing Dkt model but found this error, any solutions!!!
There is error in eval file. The auc variable is giving errors.
![WhatsApp Image 2022-11-24 at 09 47 28](https://user-images.githubusercontent.com/83466056/203809829-1d54243d-c430-488c-8b9c-1cdf29e0bff0.jpg)

**Comment by sonyawong** [comment]
> We have been testing Dkt model but found this error, any solutions!!! There is error in eval file. The auc variable is giving errors. ![WhatsApp Image 2022-11-24 at 09 47 28](https://user-images.githubusercontent.com/83466056/203809829-1d54243d-c430-488c-8b9c-1cdf29e0bff0.jpg)

Hi, it seems like the labels of the testing set only contain one class hence making the AUC score computation raise an error. You can check whether there are two classes in the testing set.
